okhttp/src/main/java/okhttp3/internal/connection/RouteDatabase.kt

```diff
@@ -13,33 +13,32 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package okhttp3.internal.connection;
+package okhttp3.internal.connection
 
-import java.util.LinkedHashSet;
-import java.util.Set;
-import okhttp3.Route;
+import okhttp3.Route
+import java.util.LinkedHashSet
 
 /**
  * A blacklist of failed routes to avoid when creating a new connection to a target address. This is
  * used so that OkHttp can learn from its mistakes: if there was a failure attempting to connect to
  * a specific IP address or proxy server, that failure is remembered and alternate routes are
  * preferred.
  */
-final class RouteDatabase {
-  private final Set<Route> failedRoutes = new LinkedHashSet<>();
+class RouteDatabase {
+  private val failedRoutes = LinkedHashSet<Route>()
```

Comment:
linkedSetOf<Route>()

Reply:
Changed to `private val failedRoutes: HashSet<Route> = linkedSetOf()`